module/Decision/src/Model/SubDecision/Board/Release.php

```diff
@@ -53,7 +54,7 @@ class Release extends SubDecision
     /**
      * Date of the discharge.
      *
-     * @ORM\Column(type="date")
+     * @Column(type="date")
```

Comment:
These are leftover from the switch to native attributes instead of the Doctrine annotations, they can and should be removed (which means that we do not need the import). The same applies to the `Album` entity.

Reply:
I fixed that